Dockerfile

```diff
@@ -10,6 +10,16 @@ RUN curl -LO https://github.com/polyml/polyml/archive/v5.8.tar.gz && \
     tar xf v5.8.tar.gz && \
     cd polyml-5.8 && ./configure --prefix=/usr && make && make install
 
+FROM ubuntu:20.04
+
+RUN apt-get update && \
+    apt-get install -y jq && \
+    apt-get purge --auto-remove && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+COPY --from=builder /usr /usr
```

Comment:
Do you need to copy the entirety of `/usr` of could you also copy just parts of it?